packages/eslint-plugin-pf-codemods/src/rules/v6/labelRemoveIsOverflowLabel/label-remove-isOverflowLabel.ts

```diff
@@ -0,0 +1,64 @@
+import { getFromPackage } from "../../helpers";
+import { Rule } from "eslint";
+import { JSXOpeningElement } from "estree-jsx";
+
+// https://github.com/patternfly/patternfly-react/pull/10037
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-core");
+
+    const labelImport = imports.find(
+      (specifier: { imported: { name: string } }) =>
+        specifier.imported.name === "Label"
+    );
+
+    return !labelImport
+      ? {}
+      : {
+          JSXOpeningElement(node: JSXOpeningElement) {
+            if (
+              node.name.type === "JSXIdentifier" &&
+              labelImport.local.name === node.name.name
+            ) {
+              const isOverflowLabelAttribute = node.attributes.find(
+                (attr) =>
+                  attr.type === "JSXAttribute" &&
+                  attr.name?.name === "isOverflowLabel"
+              );
+
+              if (isOverflowLabelAttribute) {
```

Comment:
Nit: I'd rather see this check setup as a guard clause with an early return than have the whole rest of the logic nested one level deeper. IMO that layout just keeps things a little bit simpler.